### ISSUE TITLE
Obsolete `apispec` requirements in 1.0.3 wheel's dist-info metadata

### ISSUE THREAD

**Issue by agurtovoy** [Woile/starlette-apispec#39]
According to [`pyproject.toml`](https://github.com/Woile/starlette-apispec/blob/894c3da8824361de99fa8d18e24ebc3607fd11c8/pyproject.toml#L53) and [this commit](https://github.com/Woile/starlette-apispec/commit/d9a0a6456687c7c328ca58650bbb3f3242b17a74), current `apispec` requirements are ">=1,<5". However, `starlette_apispec-1.0.3.dist-info/METADATA` in the 1.0.3 wheel downloaded from pypi says `Requires-Dist: apispec (>=1,<4)`:

<img width="946" alt="Screen Shot 2021-07-14 at 1 28 06 PM" src="https://user-images.githubusercontent.com/3394202/125673716-596071bf-b975-48a5-9294-7bb096c75907.png">

Not sure how this happened, but this prevents us from using `starlette-apispec` together with the latest `apispec` (4.7.1) (using Poetry).

**Comment by agurtovoy** [comment]
Workaround: installing directly from Github helps:

```
starlette-apispec = { git = "https://github.com/Woile/starlette-apispec.git" }
```

**Comment by Woile** [comment]
I've released `v1.0.4` 🎉 which should solve this, could you verify please?

I've also automated the pipelines so any new PR will be published automatically once merged, so feel free to send new PRs if you find any issues.

Feel free to re-open the issue if needed.

**Comment by agurtovoy** [comment]
Confirmed that 1.0.4 fixes this, thanks so much for the quick response!
